packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -52,11 +52,21 @@ export default class AnthosTenantsService {
   }
 
   getTenantPublicIPs(serviceName) {
-    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`);
+    return this.fetch(`/dedicated/anthos/tenants/${serviceName}/ips/public`, {
+      headers: {
+        Pragma: 'no-cache',
+        'X-Pagination-Mode': 'CachedObjectList-Pages',
```

Comment:
By default, only 10 items are fetched. I notice that no page size is specified so only the first 10 items will be returned each time